discord/enums.py

```diff
@@ -428,6 +430,11 @@ class AuditLogAction(Enum):
     auto_moderation_user_communication_disabled = 145
     creator_monetization_request_created = 150
     creator_monetization_terms_accepted = 151
+    onboarding_question_create = 163
+    onboarding_question_update = 164
+    onboarding_update = 165
```

Comment:
Shouldn't this be 167? https://github.com/discord/discord-api-docs/pull/6041
```suggestion
    onboarding_update = 167
```